stacktemplates/stack_master.yml.erb

```diff
@@ -1,3 +1,7 @@
+# prepend_region_alias_to_stack_names: true
+# region_aliases:
+# stack_defaults:
+#   tags:
```

Comment:
I felt the default stack_master.yml from the init function was a little too barren, so I added some extra empty (commented out) options. Is that ok?